src/Compilers/VisualBasic/Portable/Binding/ImportAliasesBinder.vb

```diff
@@ -22,8 +23,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
     ''' </summary>
     Friend Class ImportAliasesBinder
         Inherits Binder
+        Implements IImportChain
```

Comment:
>Implements IImportChain [](http://example.com/codeflow?start=8&length=23)

We do not want to expose binders this way.

Reply:
Undone.  Sorry.